spec.html

```diff
@@ -30152,12 +30247,12 @@ <h1>NonemptyClassRanges</h1>
           <h1>Runtime Semantics: CharacterRange ( _A_, _B_ )</h1>
           <p>The abstract operation CharacterRange takes two CharSet parameters _A_ and _B_ and performs the following steps:</p>
           <emu-alg>
-            1. If _A_ does not contain exactly one character or _B_ does not contain exactly one character, throw a *SyntaxError* exception.
+            1. Assert: _A_ and _B_ each contain exactly one character.
```

Comment:
Doesn't e.g. `/[\p{Hex}-\uFFFF]/u` hit this case as well? The assert seems invalid in that case. What am I missing?

Reply:
It doesn't because of https://github.com/tc39/test262/pull/1014/files/4843f049fafb6cdd61e3be011e3701cc94a9fe46#r200694312. Nothing to see here folks, move along! :)